File: sdkwork-backend-sdk-typescript/src/types/page-plus-channel-account-vo.ts
import type { PageableObject } from './pageable-object';
import type { PlusChannelAccountVO } from './plus-channel-account-vo';
import type { SortObject } from './sort-object';

export interface PagePlusChannelAccountVO {
  totalPages?: number;
  totalElements?: number;
  size?: number;
  content?: PlusChannelAccountVO[];
  number?: number;
  numberOfElements?: number;
  first?: boolean;
  last?: boolean;
  sort?: SortObject;
  pageable?: PageableObject;
  empty?: boolean;
}

File: sdkwork-backend-sdk-kotlin/src/main/kotlin/com/sdkwork/backend/PagePlusChannelAccountVO.kt
package com.sdkwork.backend

data class PagePlusChannelAccountVO(
    val totalElements: Int? = null,
    val totalPages: Int? = null,
    val size: Int? = null,
    val content: List<PlusChannelAccountVO>? = null,
    val number: Int? = null,
    val numberOfElements: Int? = null,
    val sort: SortObject? = null,
    val pageable: PageableObject? = null,
    val first: Boolean? = null,
    val last: Boolean? = null,
    val empty: Boolean? = null
)
